proxy/envoy/resources.go

```diff
@@ -410,10 +410,10 @@ type SSLContext struct {
 
 // SSLContextWithSAN definition, VerifySubjectAltName cannot be nil.
 type SSLContextWithSAN struct {
-	CertChainFile        string   `json:"cert_chain_file"`
-	PrivateKeyFile       string   `json:"private_key_file"`
+	CertChainFile        string   `json:"cert_chain_file,omitempty"`
+	PrivateKeyFile       string   `json:"private_key_file,omitempty"`
 	CaCertFile           string   `json:"ca_cert_file,omitempty"`
```

Comment:
Do you know what root CAs is Envoy using? Are they packaged with the binary or do we need to add them to the base image?